core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -319,13 +320,6 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
 
   override def getLastUpdatedTime(): Long = lastScanTime.get()
 
-  /**
-   * Split a comma separated String, filter out any empty items, and return a Sequence of strings
-   */
-  private def stringToSeq(list: String): Seq[String] = {
-    list.split(',').map(_.trim).filter(_.nonEmpty).toImmutableArraySeq
-  }
```

Comment:
This is idential with the following.

https://github.com/apache/spark/blob/79cee25d1d9a0fcc49596885a17a7ffd14868d85/common/utils/src/main/scala/org/apache/spark/util/SparkStringUtils.scala#L51-L54